mlir/lib/Dialect/LLVMIR/Transforms/DIScopeForLLVMFuncOp.cpp

```diff
@@ -31,6 +31,14 @@ static FileLineColLoc extractFileLoc(Location loc) {
     return extractFileLoc(nameLoc.getChildLoc());
   if (auto opaqueLoc = dyn_cast<OpaqueLoc>(loc))
     return extractFileLoc(opaqueLoc.getFallbackLocation());
+  if (auto fusedLoc = dyn_cast<FusedLoc>(loc)) {
+    for (auto loc : fusedLoc.getLocations()) {
```

Comment:
Is there test coverage for this?

Reply:
Good question, I think not (it may even be that the two above this is also not covered). I'll add a couple.